client/extensions/woocommerce/rest-api-client/actions.js

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
Nitpick, this is an extra space, and some of these files are missing the `@format` header

Reply:
Thanks!